specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/applicationGateway.json

```diff
@@ -452,7 +452,71 @@
         "x-ms-long-running-operation": true
       }
     },
-	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableServerVariables": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/applicationGateways/{applicationGatewayName}/backendHealthOnDemand": {
+      "post": {
+        "tags": [
+          "ApplicationGateways"
+        ],
+        "operationId": "ApplicationGateways_BackendHealthOnDemand",
```

Comment:
This looks like a `GET` operation, any reason why it's a `POST` instead?

Reply:
- Want to keep it similar to an exisiting operation - backendhealth which is a POST
- Have a request body
- Typically, we are using GET and PUT for getting and setting properties of a resource. For other operations on resource, we are using POST